src/extensions/averagewaveforms/AverageWaveformsView/fetch_average_waveforms_2.py

```diff
@@ -54,8 +56,8 @@ def fetch_average_waveform_2(snippets_h5, unit_id, visible_channel_ids):
         channel_locations0.append(channel_locations[ind, :].ravel().tolist())
 
     return dict(
-        average_waveform=average_waveform.astype(float).tolist(),
-        channel_ids=unit_waveforms_channel_ids.astype(int).tolist(),
+        average_waveform=average_waveform.astype(np.float32),
```

Comment:
We can now return numpy arrays from the hither functions, as the labbox framework takes care of the serialization on the python side, and the deserialization on the js side.